robottelo/ui/role.py

```diff
@@ -87,22 +87,46 @@ def add_permission(self, role_name, resource_type=None,
         self.click(common_locators['submit'])
 
     def get_resources(self, role_name):
+        """Fetch resources from role filters.
+
+        :param role_name: String with role name.
+        :return: List of strings with resorces names.
+        """
         self.search_and_click(role_name)
         self.click(tab_locators['roles.tab_filters'])
-        resources = self.find_elements(locators['roles.resources'])
-        return [item.text for item in resources]
+        resources = [
+            item.text for item in
```

Comment:
nitpick for the next time - prefer to use more meaningful names, e.g. `resource.text for resource in ...` rather then just 'item'